lib/Repository.js

```diff
@@ -551,6 +551,49 @@ class Repository extends Requestable {
    }
 
    /**
+    * List the deploy keys for the repository
+    * @see https://developer.github.com/v3/repos/keys/#list-deploy-keys
+    * @param {Requestable.callback} cb - will receive the list of hooks
+    * @return {Promise} - the promise for the http request
+    */
+   listKeys(cb) {
+      return this._request('GET', `/repos/${this.__fullname}/keys`, null, cb);
+   }
+
+   /**
+    * Get a deploy key for the repository
+    * @see https://developer.github.com/v3/repos/keys/#get-a-deploy-key
+    * @param {number} id - the id of the webook
```

Comment:
I think `{number|string}` is better. but I think that "united feeling" will be lost with the fix only here. Because `id` is also in other places is `{number}`. For example, `getHook`. 

only here or fix all?